server/resp/src/main/java/org/infinispan/server/resp/json/EmbeddedJsonCache.java

```diff
@@ -136,7 +136,10 @@ public CompletionStage<List<String>> type(byte[] key, byte[] path) {
    public CompletionStage<Long> del(byte[] key, byte[] path) {
       return readWriteMap.eval(key, new JsonDelFunction(path));
    }
-   public CompletionStage<List<Long>> strAppend(byte[] key, byte[] path, byte[] value, AppendType type) {
-      return readWriteMap.eval(key, new JsonAppendFunction(path, value, type));
+   public CompletionStage<List<Long>> arrAppend(byte[] key, byte[] path, List<byte[]> values) {
```

Comment:
why not writing a single method append ? with the type ? we are not going to expose this api other than redis for now, are we ?

Reply:
javadocs too

Reply:
they have two different value types: byte[] and List<byte[]>. I preferred two different names since they act on different json object